Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Synchronization/RelationInconsistenciesTestBase.cs

```diff
@@ -43,7 +46,11 @@ protected void SetIndustrialSectorInOtherTransaction (ObjectID companyID, Object
 
     protected ObjectID CreateComputerAndSetEmployeeInOtherTransaction (ObjectID employeeID)
     {
-      return RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<Computer, Employee>(employeeID, (c, e) => c.Employee = e);
+      return RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<Computer, Employee>(employeeID, (c, e) =>
```

Comment:
marked file (please move property initialization up to object load/initialization)

Reply:
Same problem as above: I cannot set properties before creating the respective `DomainObject`, which is done in RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction().

Reply:
yep, same answer, too :)